### ISSUE TITLE
Add navigation bar

### ISSUE THREAD

**Issue by ank1traj** [ank1traj/testcase-generator#14]
Adding a navigation bar to the website will make it easier for users to navigate through the website and find the information they need. The navigation bar can be placed at the top of the website and can include links to different pages on the website, such as the home page, about us page, contact us page and other relevant pages.

The navigation bar should be designed in a way that is user-friendly and easy to understand. It should be visually appealing, and the links should be organized in a logical manner. It is also important to ensure that the navigation bar is responsive and works well on all devices, including desktops, laptops, tablets, and smartphones.

In addition to the navigation bar, it may also be useful to include a search bar on the website, allowing users to quickly search for specific information they need. This can be especially useful for larger websites with a lot of content.

Overall, adding a navigation bar and search bar to the website will improve user experience and make it easier for users to find the information they need.

**Comment by Shahid-Bagwan** [comment]
Can I Help?

**Comment by ank1traj** [comment]
Hi @Shahid-Bagwan 

That sounds great! Thank you for expressing your interest in adding a navigation bar. We welcome any contributions that can enhance the user experience of our platform. Please feel free to take the issue. We look forward to seeing your contributions.

Thanks 😊

**Comment by yuvrxj-afk** [comment]
![navbar](https://user-images.githubusercontent.com/63532070/230740652-45b5c904-62b3-4315-87e4-e25474be354c.png)

I tried this , what do you think about it?

**Comment by ank1traj** [comment]
Hi @yuvrxj-afk 

Looking good. Will have to check on other pages too. Could you please try with some other pages too.

**Comment by yuvrxj-afk** [comment]
Surely ankit. Looking forward to that. 
Should we create different components? Or you want it to working like this only?

**Comment by Shahid-Bagwan** [comment]
> ![navbar](https://user-images.githubusercontent.com/63532070/230740652-45b5c904-62b3-4315-87e4-e25474be354c.png)
> 
> I tried this , what do you think about it?

I really liked this design, great work @yuvrxj-afk . @ank1traj assign him the issue, he is the perfect match.

**Comment by ank1traj** [comment]
> Surely ankit. Looking forward to that. 
> Should we create different components? Or you want it to be working like this only?

@yuvrxj-afk What do you mean by different components?

**Comment by yuvrxj-afk** [comment]
Basically to use router here.

**Comment by ank1traj** [comment]
Yes, that would be great. As adding router will make easier to use the navigation.

**Comment by ank1traj** [comment]
First version is Implemented by @yuvrxj-afk. Tested it successfully. Closing the issue.
